@types/jsb.d.ts

```diff
@@ -1415,6 +1415,88 @@ declare namespace jsb {
         showExternalOfferInformationDialog: (listener: ExternalOfferInformationDialogListener) => void;
         showInAppMessages: (params: InAppMessageParams, listener: InAppMessageResponseListener) => void;
     }
+
+    export class PlayException {
+        getMessage(): string;
+        getLocalizedMessage(): string;
+        printStackTrace(): void;
+        toString(): string;
+    }
+
+    export interface OnCanceledListener {
+        onCanceled(): void;
+    }
+
+    export interface OnCompleteListener {
+        onComplete(var1: any): void;
+    }
+
+    export interface OnFailureListener {
+        onFailure(var1: PlayException): void;
+    }
+
+    export interface OnSuccessListener {
+        onSuccess(var1: any): void;
+    }
+
+    export interface OnContinueWithListener {
+        onThen(var1: any): void;
+    }
+
+    export interface OnContinueWithTaskListener {
+        onThen(var1: any): void;
+    }
+
+    export class PlayTask {
+        public addOnCanceledListener(var1: OnCanceledListener): PlayTask;
+        public addOnCompleteListener(var1: OnCompleteListener): PlayTask;
+        public addOnFailureListener(var1: OnFailureListener): PlayTask;
+        public addOnSuccessListener(var1: OnSuccessListener): PlayTask;
+        public continueWith(var1: OnContinueWithListener): PlayTask;
+        public getResult(var1: OnSuccessListener): any;
+        public isCanceled(): boolean;
+        public isComplete(): boolean;
+        public isSuccessful(): boolean;
+    }
+    export class AuthenticationResult {
+        private constructor();
+        public isAuthenticated(): boolean;
+    }
+    export class RecallAccess {
+        private constructor();
+        public hashCode(): number;
+        public getSessionId(): string;
+        public equals(other: RecallAccess): boolean;
+    }
+    export class GamesSignInClient {
+        public isAuthenticated(): PlayTask;
+        public requestServerSideAccess(var1: string, var2: boolean): PlayTask;
+        public signIn(): PlayTask;
+    }
+    export class AchievementsClient {
+        public showAchievements(): void;
+        public incrementImmediate(var1: string, var2: number): PlayTask;
```

Comment:
Don't use `var1` , `var2` for the parameter name.

Reply:
The code in your attachment is decompiled code from `*.class`. 
It's better to reference the Google Play API documentation. I don't think the API documentation also uses `var1, var2 ...`.

Reply:
Yes, it's been fixed.